client/me/purchases/controller.jsx

```diff
@@ -156,7 +156,7 @@ export default {
 		);
 	},
 
-	list() {
+	list( context ) {
```

Comment:
> In the PHP world this behavior is called Flash Messages.

Ah, I'm familiar with the pattern from Rails. :) This would be an improvement, and may be worth considering in the context of the new global notices that we're considering. I'll open an issue for using a global notice here once this is merged.